README.md

```diff
@@ -44,19 +44,37 @@ To build the operator:
     make build
     ```
 
-## Updating Custom Resource Definitions
-
-The custom resource definitions used by the Verrazzano Operator are auto generated by the Verrazzano project `verrazzano-crd-generator`.
-The steps to include changes made to the custom resource definitions are:
-
-* Make the API changes in the `verrazzano-crd-generator` project
-* Tag a branch of `verrazzano-crd-generator`, for example `0.n`
-* Edit `go.mod` to use the newly tagged API changes.  For example `github.com/verrazzano/verrazzano-crd-generator v1.2.0`
+## Updating Dependencies
```

Comment:
Updating Dependencies -> Updating dependencies (We have decided to use sentence capitalization in the open source docs instead of title capitalization. I will re-edit this entire README after your changes go in.)

Reply:
I'll leave it as is an you can correct all section capitalization at once.

Reply:
Fixed.